lib/blockscore/util.rb

```diff
@@ -11,13 +11,13 @@ module Util
     }
 
     def parse_json!(json_obj)
-      JSON.parse(json_obj, symbolize_names: true)
+      JSON.parse(json_obj, :symbolize_names => true)
     end
 
     def parse_json(json_obj)
       parse_json! json_obj
     rescue JSON::ParserError
-      raise Error, 'An error has occurred. If this problem persists, please message support@blockscore.com.'
+      fail Error, "An error has occurred. If this problem persists, please message support@blockscore.com."
```

Comment:
Metrics/LineLength: Line is too long. [107/100]<br>Style/SignalException: Use `raise` instead of `fail` to rethrow exceptions.<br>Style/StringLiterals: Prefer single-quoted strings when you don't need string interpolation or special symbols.